openedx/features/edly/cookies.py

```diff
@@ -42,10 +42,13 @@ def delete_logged_in_edly_cookies(response):
     Returns:
         HttpResponse
     """
-    response.delete_cookie(
+    response.set_cookie(
         settings.EDLY_USER_INFO_COOKIE_NAME,
         path='/',
-        domain=settings.SESSION_COOKIE_DOMAIN
+        domain=settings.SESSION_COOKIE_DOMAIN,
+        max_age=0,
+        expires='Thu, 01-Jan-1970 00:00:00 GMT',
```

Comment:
Please verify this change first. Also please include the references where other people are using this solution.

Reply:
@zubair-arbi  I have verified the changes on Dev server. The code originally copy from the `delete_cookie` function https://github.com/django/django/blob/stable/1.7.x/django/http/response.py#L260

Reply:
@burhandodhy @zubair-arbi edx has same implementation why we need to change it?
https://github.com/edx/edx-platform/blob/70a82580fb04eac1c8ece883829e579cdd5412dc/openedx/core/djangoapps/user_authn/cookies.py#L69

Reply:
@ziafazal while debugging the API calls issue on the production, we found that the Google chrome didn't delete the `edly-user-info` cookies during the logout. This happen because of Google chrome samesite cooke policy. 

To delete the cookies we need to set the `secure` argument and the default `delete_cookie` function didn't have `secure` implementation.